cmake/Dependencies.cmake

```diff
@@ -1983,6 +1983,7 @@ if(USE_KINETO)
     string(APPEND CMAKE_CXX_FLAGS " -DLIBKINETO_NOCUPTI")
     message(STATUS "Configured Kineto (CPU)")
   else()
+    list(APPEND Caffe2_DEPENDENCY_LIBS ${CUDA_CUDART_LIBRARY})
```

Comment:
Should be `Caffe2_CUDA_DEPENDENCY_LIBS` here I guess?

Reply:
Wrong. See https://github.com/pytorch/kineto/issues/356 to see why you are wrong here.

Reply:
Thanks for the link for explanation. But the fact that cudart is in torch_cpu is still a bit awkward.